src/Request/uapi-parser.js

```diff
@@ -322,4 +341,4 @@ Parser.prototype.parse = function (xml) {
   });
 };
 
-module.exports = Parser;
+module.exports = { Parser, defaultConfig, errorsConfig };
```

Comment:
Please use
```javascript
export const defaultConfig = function defaultConfig() { ... }
export const errorsConfig = function errorsConfig() { ... }
export const Parser = function errorsConfig() { ... }
export default Parser;
```

instead of proposed construction